makefile

```diff
@@ -13,12 +13,24 @@ IMPORTS:=-Ilibdparse/src\
 
 FLAGS:=-O -gc -release -inline # keep -inline; not having it triggers an optimizer bug as pf 2.066
 
-all: $(SRC)
+all: hmod pkg
+
+hmod: bin/hmod
+
+bin/hmod: $(SRC)
 	dmd $(SRC) $(IMPORTS) $(FLAGS) -ofbin/hmod
 	rm -f bin/*.o
 
 clean:
 	rm -rf bin/
+	$(MAKE) -f makd/Makd.mak clean
 
 doc:
 	./bin/hmod src/
+
+# Packaging configuration
+pkg: $O/pkg-hmod.stamp
+	$(MAKE) -f makd/Makd.mak pkg
+
+$O/pkg-hmod.stamp: \
+	hmod
```

Comment:
This looks a bit hacky, although I guess it should work. I wonder if it's worth adding another makefile (like `pkg.mak`) that defines dependencies as usual for Makd and includes makd, so we don't have half the dependency management in this `Makefile` and the other half in `Makd.mak`.

What I suggest is basically something like:

`pkg.mak`:
```make
include `makd/Makd.mak`

$O/pkg-hmod.stamp: hmod
```

and then in this `Makefile` just do:
```make
.PHONY: pkg
pkg:
        $(MAKE) -f pkg.mak pkg
```

Reply:
All right, I don't see a reason why not to do it. I'll amend this tomorrow, otherwise feel free to push to this branch & merge.

Reply:
Hmm, I've tried this, but in order make `$O/pkg-hmod.stamp: hmod` I need to include `makefile` from the `pkg.mak` (because that's where `hmod` target is defined) and that feels even more hacky IMHO.

Reply:
Right, another alternative would be to put in the `Makefile` `pkg: hmod` but again, is not that clean anyway...